<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <listeners>
    <listener class-name="com.rahulshettyacademy.testcomponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="Error Validation Test(failed)" verbose="0">
    <classes>
      <class name="com.rahulshettyacademy.tests.ErrorValidationsTest">
        <methods>
          <include name="launchApplication"/>
          <include name="tearDown"/>
          <include name="endToEnd"/>
        </methods>
      </class> <!-- com.rahulshettyacademy.tests.ErrorValidationsTest -->
    </classes>
  </test> <!-- Error Validation Test(failed) -->
  <test thread-count="5" parallel="tests" name="Stand Alone Test(failed)" verbose="0">
    <classes>
      <class name="com.rahulshettyacademy.tests.StandAloneTest">
        <methods>
          <include name="launchApplication"/>
          <include name="submitOrder" invocation-numbers="0"/>
          <include name="tearDown"/>
          <include name="orderHistoryTest"/>
        </methods>
      </class> <!-- com.rahulshettyacademy.tests.StandAloneTest -->
    </classes>
  </test> <!-- Stand Alone Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
